server/windows/src/windows.rs

```diff
@@ -56,8 +57,14 @@ mod omsupply_service {
         let executable_path = current_exe().unwrap();
         let executable_directory = executable_path.parent().unwrap();
         set_current_dir(&executable_directory).unwrap();
-        let settings: Settings =
-            configuration::get_configuration().expect("Failed to parse configuration settings");
+        let settings: Settings = match configuration::get_configuration() {
+            Ok(settings) => settings,
+            Err(e) => {
+                eventlog::init("Application", log::Level::Error).unwrap();
```

Comment:
does this have to be instantiated here, can it be instantiated a the top of `omsupply_service_main` ?